content/en/apps/guides/forms/form-inputs.md

```diff
@@ -145,6 +145,14 @@ A value can be pre-selected for the search box via a `calculate` expression. By
 | string      | person_id    | Person ID    | select-contact type-person |                              |
 | string      | household_id | Household ID | select-contact type-clinic | ../inputs/contact/parent/_id |
 
+### Loading descendants of the current contact
+
+Use the appearance `descendant-of-current-contact` to load the current contact's descendant when opening an app form or contact form from the Contact's tab.
```

Comment:
```suggestion
Use the appearance `descendant-of-current-contact` to load the current contact's descendants when opening an app form or contact form from the "People" tab.
```
This is to make it consistent with other references to the "People" tab throughout the page.